Mixpanel/Mixpanel.m

```diff
@@ -191,6 +194,7 @@ - (instancetype)initWithToken:(NSString *)apiToken launchOptions:(NSDictionary *
 
         self.decideResponseCached = NO;
         self.showSurveyOnActive = YES;
+        self.enableABTestDesigner = YES;
```

Comment:
I've been thinking about the default value of this. It's more of a product question which should involve Nihar but I'd suggest to turn this off by default.

1. It saves battery/data usage for the users
2. Our switchboard is receiving a lot of requests which are accidentally made by the users.

Also, it'd be valuable to figure out the radio of A/B users vs SDK users.